website/www/site/layouts/partials/section-menu/en/get-started.html

```diff
@@ -19,9 +19,11 @@
   <ul class="section-nav-list">
     <li><a href="/get-started/try-apache-beam/">Try Apache Beam</a></li>
     <li><a href="/get-started/try-beam-playground/">Try Beam Playground (Beta)</a></li>
-    <li><a href="/get-started/quickstart-java/">Java quickstart</a></li>
+    <li><a href="/get-started/quickstart/java/">Java quickstart</a></li>
     <li><a href="/get-started/quickstart-py/">Python quickstart</a></li>
     <li><a href="/get-started/quickstart-go/">Go quickstart</a></li>
+    <li><a href="/get-started/quickstart-java/">WordCount quickstart for
```

Comment:
I say just wipe out the old one. Was there a decision to keep it? The use of maven archetypes is just weird and unhelpful.

Reply:
We're not quite ready to remove the old WordCount quickstarts yet. The Dataflow quickstarts are still based on the WordCount examples, so this needs to stay around a little longer.

Reply:
Leaving for now per @pcoet